book/release-service.md

```diff
@@ -39,8 +28,95 @@ The diagram below shows the flow of custom resources and the orchestration of pi
+**Bundle** - A tekton bundle (image) that contains Tekton resources
+**Development Workspace** - workspace shared by 1 or more team members when code is built and tested.
+**Managed Workspace** - workspace that is managed by an SRE or Release Engineering team that is controlled. This is where Release pipeline execute.
+
+## Resources
+Below are the list of CRs that the Release service is responsible for interacting and performing operations:
+
+### CREATE
+
+| Custom Resource            | When?                                                    | Why?                                                                                       |
+|----------------------------|----------------------------------------------------------|--------------------------------------------------------------------------------------------|
+| SnapshotEnvironmentBinding | When deploying the Snapshot to an Environment            | Create a SnapshotEnvironmentBinding for the Referenced Environment when one does not exist |
+| PipelineRuns               | Once a Release has been composed and ready for execution | To perform the steps in the Release Strategy                                               |
+
+### READ
+
+| Custom Resources        | When?                                                  | Why?                                                                     |
+|-------------------------|--------------------------------------------------------|--------------------------------------------------------------------------|
+| Application & Component | When validating ReleasePlans and ReleasePlanAdmissions | To ensure data consistency                                               |
+| ReleasePlan             | When starting to process a Release                     | To determine the target workspace to run the Release Pipeline            |
+| ReleasePlanAdmission    | When starting to process a Release                     | To determine the Release Strategy to use to compose the Release Pipeline |
+| Snapshot                | Creating and triggering a Release Pipeline             | To determine which components to release                                 |
+| ReleaseStrategy         | Before composing the Release Pipeline                  | To determine the bundle to use to run the Release Pipeline               |
+
+### UPDATE
+
+| Custom Resource            | When?                                                    | Why?                                                         |
+|----------------------------|----------------------------------------------------------|--------------------------------------------------------------|
+| SnapshotEnvironmentBinding | When deploying the Snapshot to an Environment            | Update exiting SnapshotEnvironmentBinding for a new Snapshot |
+| Release                    | During the lifecycle of an attempt to release a Snapshot | Provide status for the execution of the Release Pipeline     |
+
+### WATCH
```

Comment:
We also watch Releases... not sure if this is redundant and should be left omitted though